dev/tests/integration/testsuite/Magento/Checkout/_files/active_quote.php

```diff
@@ -9,3 +9,11 @@
     ->setIsMultiShipping(false)
```

Comment:
Please, add rollback for this fixture

Reply:
The rollback for the mentioned fixture exists (active_quote_rollback.php). As for masked id - the corresponding database entry is being removed when the quote is removed, so the current rollback works for masked ids as well (have just double checked this behavior)